timeout: 700
model_name: "turtlebot3_waffle"
amcl: False
num_tracks: 6
min_dist_to_obstacle: 0.16
min_dist_to_goal: 0.2
list_init_pose: 
    0:
      x: 5
      y: 5
    1:
      x: 6
      y: 6
list_goals:
    0:
      x: 25
      y: -5
    1:
      x: 25
      y: -5
